mod-invoice-storage/schemas/voucher.json

```diff
@@ -0,0 +1,117 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "voucher",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "UUID of this voucher",
+      "type": "string",
+      "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
+    },
+    "systemCurrency": {
+      "description": "The currency in which the voucher was paid",
+      "type": "string"
+    },
+    "invoiceCurrency": {
+      "description": "The currency in which invoice was originally defined",
+      "type": "string"
+    },
+    "exchangeRate": {
+      "description": "Exchange rate used to generate transaction",
+      "type": "number"
+    },
+    "exportToAccounting": {
+      "description": "Export this payment record to a financial system",
+      "type": "boolean"
+    },
+    "voucherType":{
+      "description": "Indicates type of payment to account",
+      "type": "string",
+      "enum": [
+        "Payment",
+        "Pre-Payment",
+        "Credit",
+        "Voucher"
+      ]
+    },
+    "voucherStatus":{
+      "description": "status of the voucher",
+      "type": "string",
+      "enum": [
+        "Awaiting Payment",
+        "Paid"
+      ]
+    },
+    "voucherAmount": {
+      "description": "The amount that is actually paid",
+      "type": "number"
+    },
+    "voucherDate": {
```

Comment:
Will this date be auto-generated? If it is then this field should be read-only?

Reply:
For now we do not have all the details for this field, so probably can leave it for now and change it when we have more details

Reply:
yeah, leave as-is for now please

Reply:
Yes, please leave as-is for now.